.github/workflows/ci.yml

```
@@ -38,6 +38,14 @@ jobs:
           python-version: '3.x'
           cache: 'pip'
 
+      - name: Setup Nu
+        uses: hustcer/setup-nu@main
+        with:
+          version: '0.66.2'
+
+      - name: Update nuon
```

```suggestion
      - name: Update the repo cache
```

How about more descriptive name?

---

especially because it's not even a nuon now iirc